src/server.ts

```diff
@@ -0,0 +1,23 @@
+import "config/env";
+import express, { Router } from "express";
+import { getLogger } from "./config/logger";
+import { setupGithubWebhooks } from "./webhook/setupGithubWebhooks";
+import { webhookReceiver } from "./webhook/webhook-receiver";
+import { Webhooks } from "./webhook/webhooks";
+const app = express();
+const port = 8082; 
+
+const logger =  getLogger("server");
+
+const webhooks = new Webhooks();
+setupGithubWebhooks(webhooks);
+
+const RootRouter = Router();
+RootRouter.post("/webhook/event/:uuid", webhookReceiver.bind(null, webhooks));
```

Comment:
Why are you using `bind` here?

Reply:
The idea here is to webhookReceiver also get webhooks  as parameter along with request and response. The webhooks holds information about subscribed events and their handlers.

Reply:
I'm wondering why are we not using the existing router that all other endpoints are using?